### DISCUSSION TITLE
Skipping OS updates

### DISCUSSION THREAD

**Discussion by mwright-git** [home-assistant/operating-system#1820] [Q&A]
I have not been paying attention to OS updates recently and I have just realised that I am currently running 6.5 with 7.5 being available.

Does anyone see any problem doing that upgrade? Are there any issues I should be aware of skipping that many versions?

THis is running on a Raspberry PI 4 by the way.

Thanks

**Comment by agners** [comment]
That is being done quite often without problems. Updating to the next major version should work without problems. 

Also, we have a A/B (backup system) boot scenario, so even in the case that the new OS is not bootable somehow, it should revert back automatically. We do have restrictions on downgrades sometimes. But it shouldn't lead to unbootable systems.
